modules/ROOT/pages/gathering-set-up-information.adoc

```diff
@@ -0,0 +1,54 @@
+= Gathering Setup Information for SSO
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the following sections to gather the information you'll need to set up SSO.
+
+You might need to collaborate with your security team or other team members if you don't have access to this information. For more information, see xref:user-roles-permissions-and-profiles.adoc#user-roles-for-configuring-sso[User Roles for Configuring SSO].
+
+[[username-strategy]]
+== Username Assignment Strategy for a Shared Identity
+When an identity provider is already configured for your Anypoint Platform organization, review the username assignment strategy since it can be different for the portal and Anypoint Platform users. 
+
+To have a single identity between the portal and Anypoint Platform, the format for the username assignment must be the same. If the username strategy does not match, you must reconcile the username format by adding attributes for *anypoint_idp_id* and *anypoint_username*. Then, this information must be mapped with the Anypoint Platform identity provider that you are using for SSO. 
+
+For more information about adding custom claims when the identity provider is already configured but not in the required format, see the *Required Claims and Attributes for SSO* section. 
```

Comment:
Re: the *Required Claims and Attributes for SSO* section

Is this a section in the UI or in the doc? If doc, suggest "see _Required Claims and Attributes for SSO_. Or make it a link.